beacon.yaml

```diff
@@ -3175,41 +3320,78 @@ components:
           example: "BAM format"
 
     FilteringTermsResponse:
-      type: object
       description: |
-        Filtering terms available in this Beacon.
+        Response of a query over Beacon info. Use `InfoResponseContent` when querying the Beacon info endpoints, and `DatasetResponseContent` when querying the datasets endpoint.
```

Comment:
I think this description is wrong ;)

Reply:
definitely seems wrong to me too so I reverted the change.